infermedica_api/connectors/__init__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+"""
+infermedica_api.connectors
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This module contains API Connector classes and related utils.
+"""
+from typing import Union
+
+from .common import SearchConceptType
+from .v2 import BasicAPIv2Connector, APIv2Connector, ModelAPIv2Connector
```

Comment:
Q: Why the difference where v3 has no "model"?

Reply:
Two reasons 1) extra time is needed 2) it is a questionable feature -- so if needed we will implement it later on.